src/community/graticule/src/main/java/org/geotools/data/graticule/gridsupport/LineFeatureBuilder.java

```diff
@@ -0,0 +1,110 @@
+package org.geotools.data.graticule.gridsupport;
+
+import java.text.Format;
+import java.util.Map;
+import org.geotools.api.feature.simple.SimpleFeatureType;
+import org.geotools.api.referencing.crs.CoordinateReferenceSystem;
+import org.geotools.api.referencing.crs.ProjectedCRS;
+import org.geotools.api.referencing.cs.CoordinateSystem;
+import org.geotools.grid.GridElement;
+import org.geotools.grid.GridFeatureBuilder;
+import org.geotools.grid.ortholine.LineOrientation;
+import org.geotools.grid.ortholine.OrthoLine;
+import org.geotools.referencing.CoordinateFormat;
+import org.locationtech.jts.geom.Coordinate;
+
+public class LineFeatureBuilder extends GridFeatureBuilder {
```

Comment:
Also code from this class seems to be not in use, besides the set of constants at the top. If that's the case, I'd swap the constants around and remove the class.